cfgov/alerts/github_alert.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import github3
+
+
+class GithubAlert(object):
+    def __init__(self, credentials):
```

Comment:
Minor suggestion: what about passing the credentials as kwargs, i.e. `def __init__(self, **credentials)`? This wouldn't require changing any code below, but would clean up usage of this class a bit, allowing you to do `GithubAlert()` instead of `GithubAlert({})`?

Similar comment applies to `MattermostAlert`.

Reply:
I like this idea and may address it in a separate PR.